### PR TITLE
Replace pink with blue

### LINKED ISSUE #14027: [Colors] Replace usages of Pink with equivalent shades of Blue
To ensure that we are up to date with the latest best practices of the [Color Studio palette](https://github.com/Automattic/color-studio/blob/master/CHANGELOG.md), we should replace our current uses of Pink (for links, buttons, etc) with the equivalent shade of the new blue. It should just be a case of replacing it with the equivalent numbered shade of Blue.

We should also take this opportunity to update our color palette asset catalog to use the latest colors. The Color Studio repo contains a builder script that should automatically XML.

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #14027 

This PR replaces the pink color with blue in most of the app. This is done by:
- Replacing colorSecondary value with blue
- Replacing colorSecondaryVariant value with blue
- In the rest of the places I've manually updated the values where it makes sense.

I was considering another approach - instead of changing the value of colorSecondary replacing colorSecondary in code with colorPrimary. In the end I've decided to not follow this approach because keeping both primary and secondary set allows us to easily change the value in the future (if we ever decide the secondary color is still required). 

Exceptions to this change are:
- Stats bar chart - the accent still needs to be pink
- The bottom bar notifications badge - since the icon is blue, the badge remains pink

To test: 
- Go through the main flows - Login, Create a site, Quick start
- Check the buttons and the links are all blue now
- Check nothing is out of place
- Be cautious about dialogs

To test 2:
- Go to stats with a lot of data
- Go to Year view
- Notice the selected day in the bar chart is still pink
- Scroll to the Countries block
- Notice the countries are now in blue

To test 3: 
- Go to a site with a new notification
- Notice the bottom bar notifications badge is pink

**Updated countries map** 
map

**Bar chart with pink selected bars**
bar_chart

**Blue dialog buttons**
dialog_buttons

**Updated signup button**
signup_button

**Notifications badge still pink with blue FAB**
notifications_badge

## Regression Notes
1. Potential unintended areas of impact
- the whole app

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- I went through the login/create a site flow, smoke tested stats, notifications, comments and whatever I could think of

3. What automated tests I added (or what prevented me from doing so)
- color changes shouldn't be automatically tested

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by mattmiklic — APPROVED** [review]
Hi @planarvoid I spent some time browsing through the app and the color changes all look good to me so far. I've asked the other designers to review it as well, but any improvements we suggest can be tackled in separate issues.

**Review by renanferrari — APPROVED** [review]
LGTM 👍

Feel free to merge this once the design folks approve it.

**Comment by renanferrari** [comment]
Merging this now, as it is targeting `17.1`. I believe we can handle any further reviews from the design folks in separate PRs, as previously suggested by @mattmiklic.

**Comment by osullivanchris** [comment]
Taking a quick look at this and it looks good from my side 👍 👌

**Pull request merged**
